### PR TITLE
fixing error when there isn't both js and sass in a component

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

File: lib/tasks/build.js [hunk 1]
```diff
@@ -14,10 +14,13 @@ var files = require('../helpers/files');
 var log = require('../helpers/log');
 
 module.exports = function(gulp, config) {
-	return merge(
-		module.exports.js(gulp, config),
-		module.exports.sass(gulp, config)
-	);
+	var jsStream = module.exports.js(gulp, config);
+	var sassStream = module.exports.sass(gulp, config);
+	if(jsStream && sassStream){
```
Comment (AlbertoElias) [inline lib/tasks/build.js]: Would you mind adding a space surrounding the brackets for style consistency?

Comment (aintgoin2goa) [inline lib/tasks/build.js]: sorry what do you mean?

Comment (AlbertoElias) [inline lib/tasks/build.js]: `if (jsStream && sassStream) {` 

That

**Comment by aintgoin2goa** [comment]
I'm guessing this is fixed in truck anyway - do we still need this pull request or shall I close it?

**Comment by AlbertoElias** [comment]
Why do you think it's fixed in master?

**Comment by aintgoin2goa** [comment]
Oh, it's not.  In that case, we really need to get this fix in as it is currently impossible to build a component that is only sass or only js

**Pull request merged**